Source/Core/AudioCommon/WaveFile.cpp

```diff
@@ -26,7 +27,7 @@ WaveFileWriter::~WaveFileWriter()
   Stop();
 }
 
-bool WaveFileWriter::Start(const std::string& filename, unsigned int HLESampleRate)
+bool WaveFileWriter::Start(const std::string& filename, unsigned int HLESampleRateDivisor)
```

Comment:
Since you're renaming this variable anyway, could you change it so it matches our style guide? (`hle_sample_rate_divisor`)